CONTRIBUTING.md

```diff
@@ -0,0 +1,18 @@
+## Contributing
+
+All pull requests should be made to the develop branch.
+
+```
+git clone https://github.com/vinnymac/PokeNurse
+cd PokeNurse
+git checkout develop
+yarn
```

Comment:
might want to add a note about installing yarn like `npm i -g yarn` or a link to the yarn website install instructions.